src/features/homepage/Homepage.js

```diff
@@ -0,0 +1,84 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import {
+  BsGraphUp, BsFillBarChartLineFill, BsFillCalculatorFill,
+  BsFillCalendar2DateFill, BsFillPieChartFill, BsFillRssFill,
+  BsChatQuoteFill, BsCoin, BsCpuFill, BsGraphDown, BsTerminalFill,
+  BsMic, BsGear, BsChevronLeft,
+} from 'react-icons/bs';
+import {
+  RiAtLine, RiCloudLine, RiCodeBoxFill, RiBaseStationLine,
+  RiComputerLine, RiRadarFill, RiSave3Fill, RiWifiFill,
+} from 'react-icons/ri';
+import { selectAllQuotes, selectState, fetchQoutes } from './homepageSlice';
+import Card from './Card';
+import './css/Homepage.css';
+
+// import headerImg from '../../assets/images/homeHeader1.png';
+
+const Homepage = () => {
+  const dispatch = useDispatch();
+  const status = useSelector(selectState);
+  const contents = useSelector(selectAllQuotes);
+  useEffect(() => {
+    if (contents.length === 0 && status === 'idle') {
+      dispatch(fetchQoutes());
+    }
+  });
+  const quotes = useSelector(selectAllQuotes);
+  return (
+    <main className="homepage">
+      <nav id="home-nav">
+        <BsChevronLeft className="navIcon" />
+        Tech Stock
+        <div id="rightNav">
+          <BsMic className="navIcon" />
+          <BsGear className="navIcon" />
+        </div>
+      </nav>
+      <header id="home-header">
+        {/* <img id="home-img" src={headerImg} alt="stock-market" /> */}
+        <div className="headerImg">
```

Comment:
- [ ] Let's try not to have commented-on code in our codebase as they make our code dirty and not look professional. It's a good convention to not leave unused codes commented in our codebase. Please remove all commented codes from your codebase 👍🏽